src/app/beer_garden/db/mongo/api.py

```diff
@@ -388,3 +388,25 @@ def replace_commands(
     mongo_system.save()
 
     return to_brewtils(mongo_system)
+
+
+def distinct_namespaces() -> List:
+    namespaces = list()
+
+    single_namespace_models = [brewtils.models.Request, brewtils.models.System]
+
+    list_namespace_models = [brewtils.models.Garden]
+
+    for model_class in single_namespace_models:
+        model = _model_map[model_class].objects
+        for namespace in model.distinct("namespace"):
+            if namespace not in namespaces:
+                namespaces.append(namespace)
+
+    for model_class in list_namespace_models:
+        model = _model_map[model_class].objects
+        for namespace in model.distinct("namespaces"):
+            if namespace not in namespaces:
+                namespaces.append(namespace)
+
+    return namespaces
```

Comment:
I can see the need to expose a `distinct` function in the db api, but I think the namespace logic should live in namespace.py.

Reply:
This is fixed in #435